packages/manager/modules/storages/src/netapp/share/shares.controller.js

```diff
@@ -0,0 +1,57 @@
+import snakeCase from 'lodash/snakeCase';
+import reduce from 'lodash/reduce';
+
+export default class NetappShareCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+
+    this.shareStateEnum = [
+      'creating',
+      'available',
+      'deleting',
+      'deleted',
+      'error',
+      'inactive',
```

Comment:
could this enum be fetched from the API schema instead ?

Reply:
Yes I agree with that but in this iteration, the API schema did not include those information